app/scripts/components/common/mapbox/layer-legend.tsx

```diff
@@ -338,9 +343,9 @@ export function LayerGradientGraphic(props: LayerLegendGradient) {
       </dt>
       <dd>
         <span>{printLegendVal(min)}</span>
-        {unit?.label && <span className='unit'>{unit.label}</span>}
         <span>{printLegendVal(max)}</span>
       </dd>
+      {unit?.label && <span className='unit'>{unit.label}</span>}
```

Comment:
`<dd>` tag seems to fit more to what it does.

Reply:
@hanbyul-here - I made that change and removed the block on line 161 that was no longer targeting the units